application/src/test/java/org/togetherjava/tjbot/commands/utils/MessageUtilsTest.java

```diff
@@ -44,7 +45,118 @@ void escapeMarkdown() {
                         ```java
                         "Hello \\" World\\\\\\"" haha
                         ```
-                        """));
+                        """),
+                new TestCase("escape real example",
+                        """
+                                Graph traversal can be accomplished easily using \\*\\*BFS\\*\\* or \\*\\*DFS\\*\\*. The algorithms only differ in the order in which nodes are visited: https://i.imgur.com/n9WrkQG.png
```

Comment:
Indent this like every other test case, please.

Reply:
Spotless likes to disagree with you 😆